proto/beacon/p2p/v1/types.proto

```diff
@@ -86,21 +86,32 @@ message AttestationDataAndCustodyBit {
 }
 
 message Validator {
-  bytes pubkey = 1;
-  // TODO(781): The usage of withdrawal_credentials is not defined in spec. Not used in Prysm yet.
-  bytes withdrawal_credentials_hash32 = 2;
-  uint64 activation_epoch = 5;
-  uint64 exit_epoch = 6;
-  uint64 withdrawal_epoch = 7;
-  uint64 slashed_epoch = 8;
+  // BLS public key
```

Comment:
Many of these comments don't add additional information when the property name is descriptive enough.

Please use comments sparingly and consider removing redundant information.